test/powershell/Modules/Microsoft.PowerShell.Management/UnixStat.Tests.ps1

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+Describe "UnixFileSystem additions" -Tag "CI" {
+    # if PSUnixFileStat is converted from an experimental feature, these tests will need to be changed
+    BeforeAll {
+        $experimentalFeatureName = "PSUnixFileStat"
+        $skipTest = -not $EnabledExperimentalFeatures.Contains($experimentalFeatureName)
```

Comment:
Consider setting PSDefaultParameterValue for `Skip`

Reply:
done